README.md

```diff
@@ -100,6 +100,37 @@ To fix this, run the script:
 yarn build:local-packages
 ```
 
+#### WINDOWS Installation Notes
+
+ scoop tool installation:	// involves some powershell env setups
+  https://github.com/lukesampson/scoop/wiki/Quick-Start
+
+```
+ scoop install -g yarn
+ scoop install -g nodejs
+ yarn --version			// 1.22.4 in May/June 2020
+ node --version			// v14.4.0 on June 3, 2020
+```
```

Comment:
Were there directions for installing `yarn` and `nodejs` elsewhere we could link to?

Reply:
2cents: We could put links to the package.json here instead of updating the readme with every version bump